ktor-client/ktor-client-plugins/ktor-client-logging/common/src/io/ktor/client/plugins/logging/Logging.kt

```diff
@@ -165,6 +514,23 @@ public val Logging: ClientPlugin<LoggingConfig> = createClientPlugin("Logging",
             return@on
         }
 
+        if (okHttpFormat) {
```

Comment:
it would be nice to have extensions for okHttp format grouped in a separate file